tests/foreman/cli/test_ansible.py

```diff
@@ -510,8 +510,12 @@ def test_positive_run_packages_and_services_job(
         result = client.execute(f'systemctl status {service}')
         assert result.status == 0
 
+    @pytest.mark.upgrade
+    @pytest.mark.no_containers
     @pytest.mark.rhel_ver_list([8])
```

Comment:
```suggestion
    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
```

Reply:
I'm think to parametrize this test on 8 and 9, instead of keeping this to default

Reply:
Are you going to parameterize it?

Reply:
Not now, I figured it doesn't matter to parametrize this on multiple versions

Reply:
Updated as suggested to run on default version